Updated installation.rst for SciPy 2021 (#285)
holoviz/holoviz: doc/installation.rst

Step 1: Install a `Miniconda <https://conda.io/miniconda.html>`_  (or `Anaconda <https://www.anaconda.com/downloads>`_) environment
-----------------------------------------------------------------------------------------------------------------------------------

Any Linux, Mac OS X, or Windows computer with a web browser (preferably
Google Chrome) should be suitable. 16GB of RAM is required for some of
the examples, but most will run fine in 4GB.

If you don't already have conda on your machine, you can get it from
`conda.io <https://conda.io/miniconda.html>`_, and then open a terminal

> window.

Reviewer: ```suggestion
window with the conda environment activated.
```

If you do have conda already, it's a good idea to update it (running it
twice to get the very latest) to ensure you have the latest version::

   > conda update conda
   > conda update conda

You should then make sure you have `anaconda-project` installed after
activating your chosen conda environment:

::

   > conda install anaconda-project

Step 2: Download and extract the tutorial project
-------------------------------------------------

The following steps will fetch and extract the latest version of the

> tutorial project. *If you are attending a live tutorial or workshop,

Reviewer: ```suggestion
tutorial project. The tutorial will be installed in a new conda environment
in this subdirectory, and will not affect your current environment.
*If you are attending a live tutorial or workshop,
```

make sure to run these steps again the day before the tutorial to ensure
you have the same version of the project that the presenter will use.*

::

   > anaconda-project download pyviz/holoviz_tutorial

If you have already run this command in the current directory, you may
need to remove the existing `holoviz_tutorial` directory before the
command above will execute. Once the download is completed, change to
the tutorial directory with:

::

     > cd holoviz_tutorial

Step 3: Option A: Launch Jupyter Notebook
-----------------------------------------

You can then launch the classic Jupyter notebook server and client::

   > anaconda-project run

> A browser window with a Jupyter Notebook instance should now open,

Reviewer: ```suggestion
A browser tab with a Jupyter Notebook instance should now open,
```

Step 1: Install a `Miniconda <https://conda.io/miniconda.html>`_  (or `Anaconda <https://www.anaconda.com/downloads>`_) environment
-----------------------------------------------------------------------------------------------------------------------------------

Any Linux, Mac OS X, or Windows computer with a web browser (preferably
Google Chrome) should be suitable. 16GB of RAM is required for some of
the examples, but most will run fine in 4GB.

If you don't already have conda on your machine, you can get it from
`conda.io <https://conda.io/miniconda.html>`_, and then open a terminal
window.

If you do have conda already, it's a good idea to update it (running it
twice to get the very latest) to ensure you have the latest version::

   > conda update conda
   > conda update conda

You should then make sure you have `anaconda-project` installed after

> activating your chosen conda environment:

Reviewer: "activating your chosen envt" is vague, given that these are instructions for new users. Which envt do I need?, they might wonder.  How do you activate it?. We could put in a link to the conda docs, but here we should probably just say that the remaining commands should be executed in a conda environment.  We should also emphasize that none of these commands will _change_ their conda environment, i.e. clarify that the result will be a new conda envt just for the tutorial.

Author: > ... that none of these commands will change their conda environment

Adding `anaconda-project` may well change their conda environment, but not radically. All the steps afterwards are isolated though.

I agree with your suggestion though but is hard to phrase sensibly without potentially confusing people...

::

   > conda install anaconda-project

Step 2: Download and extract the tutorial project
-------------------------------------------------

The following steps will fetch and extract the latest version of the
tutorial project. *SciPy 2021 tutorial attendees should make sure to run
these steps the day before the tutorial to ensure they have the latest

> version of the project.*

Reviewer: Maybe "If you are attending a live tutorial or workshop, make sure to run these steps again the day before the tutorial to ensure  you have the same version of the project that the presenter will use.". Otherwise we'll have to update this page later!

Author: Applied suggestion in 2ae098f

::

   > anaconda-project download pyviz/holoviz_tutorial

If you have already run this command in the current directory, you may
need to remove the existing `holoviz_tutorial` directory before the
command above will execute. Once the download is completed, change to
the tutorial directory with:

::

     > cd holoviz_tutorial

Step 3: Option A: Launch Jupyter Notebook
-----------------------------------------

> You can then launch the classic notebook server and client::

Reviewer: ```suggestion
You can then launch the classic Jupyter notebook server and client::
```

   > anaconda-project run

A browser window with a Jupyter Notebook instance should now open,

> letting you select and execute each notebook.  You can start with the
> ones in the ``tutorial`` subdirectory, as these show how to use the
> others in ``tutorial/exercises``.  The first notebook to load is
> ``index.ipynb`` in ``tutorial/``, which makes it simple to subsequently
> load each of the others.

Reviewer: ```suggestion
letting you navigate through subdirectories and select a notebook to work on.
In this case, look in the  ``tutorial`` subdirectory, and load ``index.ipynb``, which will let you launch each of the tutorials and exercises.
```

> If you don't see the notebook appear, you may need to cut and paste the
> URL from the console output manually.

Reviewer: ```suggestion
If you don't see the notebook appear as a new browser tab automatically, you 
may need to cut and paste the URL from the console output manually.
```

Step 3: Option B: Launch JupyterLab
-----------------------------------

> If you prefer to use JupyterLab, you can use the following command instead::

Reviewer: ```suggestion
If you prefer to use JupyterLab rather than classic (single-page) Jupyter, you can use the following command instead::
```
